functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,26 @@
  */
 struct BeerOrganizer
 {
+    BeerBrand operator()()
+    {
+        constexpr size_t BEER_NUM = 6;
+        constexpr wall_shelf<BEER_NUM> brands {
```

Comment:
Still the problem of possible 50+ values and their initialization :)
I would prefer to iterate over enum, with range specification.
Also, what if the original BeerBrand enumeration will be changed?